flow/flow.go

```diff
@@ -463,9 +463,11 @@ func (f *Flow) updateMetricsWithLinkLayer(packet *gopacket.Packet, length int64)
 	}
 
 	if f.XXX_state.link1stPacket == 0 {
-		f.XXX_state.link1stPacket = (*packet).Metadata().Timestamp.UnixNano()
-	} else {
-		if (f.RTT == 0) && (f.Link.A == ethernetPacket.DstMAC.String()) {
+		if f.Link.A == ethernetPacket.SrcMAC.String() {
```

Comment:
I think this commit is not needed as this is the first packet according to the line 465 and we always set Link.A as Src.